app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AuthenticationController < ApplicationController
+  skip_before_action :authenticate_request
+
+  # POST /auth/login
+  def login
+    @user = User.find_by_email(params[:email])
```

Comment:
Here `find_by` will not raise exception if user with given email not present.
And if given email is not present in DB then we should show `Email not found `error and **not** `Unauthorized` error
So you should use find query and if user not found then add that case and if user email is present then after that you can check token for authorization